hash/_wasm/src/lib.rs

```diff
@@ -5,48 +5,48 @@ use wasm_bindgen::prelude::*;
 
 #[wasm_bindgen]
 pub struct DenoHash {
-  inner: Box<dyn DynDigest>,
+    inner: Box<dyn DynDigest>,
```

Comment:
These changes are the result of `cargo fmt`

Reply:
we don't use `cargo fmt` in CI for this repo, I'd just leave it as is for now -- we can address formatting of rust in a separate PR (we typically indent using 2-character wide & spaces)

Reply:
reverted